jobsAdmin/init.go

```diff
@@ -521,6 +527,9 @@ func resurrectJobSummary(jm ste.IJobMgr) common.ListJobSummaryResponse {
 		jpp := jpm.Plan()
 		js.CompleteJobOrdered = js.CompleteJobOrdered || jpp.IsFinalPart
 		js.TotalTransfers += jpp.NumTransfers
+		if js.TotalTransfers > uint32(common.GetRecommendedMaxObjectsPerJob()) {
```

Comment:
constant numerics generally will type morph into the type they're used alongside